src/Classes/BaseIncome.php

```diff
@@ -10,9 +10,15 @@
     const WITHHELD = true;
 
     abstract public function getTaxBrackets();
+    abstract public function isUserClaimingExemption();
 
     public function compute()
     {
+        if ($this->isUserClaimingExemption())
+        {
+            return round(0, 2);    
```

Comment:
This... looks a little odd? What's this doing?

Reply:
:+1: